Source/WebKit/Shared/API/Cocoa/WKMain.h

```diff
@@ -35,6 +35,7 @@ extern "C" {
 WK_EXPORT int WKXPCServiceMain(int argc, const char** argv) WK_API_AVAILABLE(macos(10.15), ios(13.0));
 WK_EXPORT int WKAdAttributionDaemonMain(int argc, const char** argv) WK_API_AVAILABLE(macos(13.0), ios(16.0));
 WK_EXPORT int WKWebPushDaemonMain(int argc, char** argv) WK_API_AVAILABLE(macos(13.0), ios(16.0));
+WK_EXPORT int WKWebPushToolMain(int argc, char** argv) WK_API_AVAILABLE(macos(14.0), ios(17.0));
```

Comment:
Should this use the `_TBA` macros instead?

Reply:
Searching through the codebase, nope that ship has sailed. (I think it sails when dev betas start going out)

Reply:
But surely at this point it's wrong to say this will be available in macOS 14 and iOS 17

Reply:
Yes surely!